grammar-error-corrector/backfill.py

```diff
@@ -4,10 +4,11 @@
 
 from config import logger, GantryConfig, DataStorageConfig
 import gantry
-import gantry.query as gantry_query
+from gantry.query.time_window import TimeWindow
 
 gantry.init(api_key=GantryConfig.GANTRY_API_KEY, send_in_background=False)
-
+# application = gantry.create_application(GantryConfig.GANTRY_APP_NAME)
```

Comment:
Is it possible to do try/except here to try create_application, if it errors (application is already there), then we call `get_application`?

Reply:
Then user wouldn't have to comment/uncomment this line while running it again

Reply:
fixed! thank you